polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -1634,8 +1657,43 @@ struct ConvertPolygeistToLLVMPass
         if (failed(converter.convertTypes(op->getOperandTypes(),
                                           convertedOperandTypes)))
           return std::nullopt;
+
+        // With opaque pointers, type attributes also might need to be
+        // translated to the corresponding LLVM types, for example the element
+        // type attribute of GEP or alloca.
+        std::optional<bool> noTyAttrConversion;
+        if (useOpaquePointers)
+          noTyAttrConversion = std::transform_reduce(
+              op->getAttrs().begin(), op->getAttrs().end(),
+              std::optional<bool>{true},
+              [](std::optional<bool> b1,
+                 std::optional<bool> b2) -> std::optional<bool> {
+                if (!b1.has_value() || !b2.has_value()) {
```

Comment:
[nit]: single stmt here is braced while the single stmt at line 1665 is not braced. I would do the opposite.